atomium/includes/alter/views.inc

```diff
@@ -9,14 +9,15 @@
  * Implements hook_views_pre_render().
  */
 function atomium_views_pre_render(view $view) {
+  $extend = array();
   $display = $view->display[$view->current_display];
-  $style_plugin = $view->style_plugin->definition['theme'];
 
-  $extend = array(
-    $style_plugin,
-    $view->name,
-    $display->display_plugin,
-  );
+  if (isset($view->style_plugin->definition['theme'])) {
```

Comment:
```suggestion
  if (true === array_key_exists('theme', $view->style_plugin->definition)) {
```

Reply:
After reading [this](https://coderwall.com/p/q9erfw/isset-vs-array_key_exists), I don't really understand the reason of your change but it is OK for me.

Reply:
The main difference between `isset()` and `array_key_exists()` function is that the `array_key_exists()` function will definitely tells if a key exists in an array, whereas `isset()` will only return true if the key/variable exists and is not null.
In our case, we do not need to check if it's null, it's not up to us to verify that. We suppose that if this variable is set, it's set with a value, ready to use, and this is why we use `array_key_exists()`.